docs/content/latest/manage/backup-restore/point-in-time-restore-ycql.md

```diff
@@ -276,13 +317,192 @@ Relative times can be in any of the following formats (again, note that you can
+
+    ```output
+     employee_no | name           | department | salary
+    -------------+----------------+------------+--------
+            1223 |   Lucille Ball | Operations |  70000
+            1224 | John Zimmerman |      Sales |  60000
+            1221 |     John Smith |  Marketing |  50000
+            1222 |    Bette Davis |      Sales |  55000
+
+    (4 rows)
+    ```
+
+### Undo index creation
+
+1. Create an index on the table from the previous examples.
+
+    ```sql
+    use pitr;
+    create index t1_index on employees (employee_no);
+    describe index t1_index;
+    ```
+
+    ```output
+    CREATE INDEX t1_index ON pitr.employees (employee_no)
+    WITH transactions = {'enabled': 'true'};
+    ```
+
+1. Now restore back to a time before this index was created.
+
+    **Note**: Due to a ycqlsh caching issue, to see the effect of this change, you will need to drop out of your current ycqlsh session and log back in.
+
+    ```sh
+    ./bin/ycqlsh -e 'use pitr; describe table pitr.t1_index;'
+    ```
+
+    ```output
+    <stdin>:1:Column family u't1_index' not found
+    ```
+
+### Undo table and index deletion
+
+1. Create a second table called `dont_deleteme`.
+
+    ```sql
+    create table dont_deleteme (
+      oops integer PRIMARY KEY,
+      mistake text
+    ) with transactions = { 'enabled' : true };
+    ```
+
+1. Create an index on the table from the previous examples.
+
+    ```sql
+    use pitr;
+    create index t1_index on employees (employee_no);
+    describe index t1_index;
+    ```
+
+    ```output
+    CREATE INDEX t1_index ON pitr.employees (employee_no)
+    WITH transactions = {'enabled': 'true'};
+    ```
+
+1. Get a timestamp using one of the methods in [Restore from an absolute time](#restore-from-an-absolute-time).
+
+1. Delete the new table and index.
+
+    ```sql
+    drop table dont_deleteme;
+    drop index t1_index;
+    ```
+
+1. Restore to the timestamp.
+
+    ```sh
+    $ bin/yb-admin restore_snapshot_schedule 0e4ceb83-fe3d-43da-83c3-013a8ef592ca 1620418801439626
+    ```
+
 ## Limitations
 
-This is a BETA feature, and is in active development. Currently, you can **restore data only**. The feature doesn't support metadata; in other words, rolling back past operations such as CREATE, ALTER, TRUNCATE, and DROP TABLE is unsupported.
+This is a BETA feature, and is in active development. Currently, you can recover from the following YCQL operations:
+
+* Data changes
+* CREATE and DROP TABLE
+* ALTER TABLE (including ADD and DROP COLUMN)
+* CREATE and DROP INDEX
 
 Development for this feature is tracked in [issue 7120](https://github.com/yugabyte/yugabyte-db/issues/7120). Some forthcoming features include:
 
 * Automatic configuration
-* Support for undoing metadata operations, such as CREATE, ALTER, TRUNCATE, or DROP TABLE
-* Schedules to take snapshots at user-defined intervals
 * Options to restore with different granularities, such as a single YSQL database or the whole YCQL dataset.
```

Comment:
this is also basically done -- with the schedule taking on YCQL keyspace / YSQL database